automatic/gitextensions/gitextensions.nuspec

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>gitextensions</id>
+    <version>2.51.02</version>
+    <title>Git Extensions</title>
+    <authors>Henk Westhuis</authors>
+    <owners>chocolatey</owners>
```

Comment:
Return original maintainer, ferventcoder after the chocolatey user, comma separated.

Reply:
why? he doesn't seem to care about this anymore. or this tag should contain all the people of have contributed to this package?

Reply:
He is still the official maintainer of the package, also it isn't because he doesn't care, but rather because he doesn't have the time to maintain the packages anymore.

IMO, the tag should contain all officially listed maintainers (current and previous).

Reply:
IMHO, he should step down as the package maintainer, as in practice, he does not maintain it anymore (regardless of the reasons).

Reply:
> Also it isn't because he doesn't care, but rather because he doesn't have the time to maintain the packages anymore.

I prefer to call things as they is. So 'not care' is good in my book. 


However, the list contains previous owners no matter their involvement. Why ? Because some owners want that, and we need a rule.